process/sync/storageBootstrap/metaStorageBootstrapper.go

```diff
@@ -70,7 +70,7 @@ func (msb *metaStorageBootstrapper) applyNotarizedBlocks(
 }
 
 func (msb *metaStorageBootstrapper) getHeader(hash []byte) (data.HeaderHandler, error) {
-	return msb.getMetaHeaderFromStorage(hash)
+	return process.GetMetaHeaderFromStorage(hash, msb.marshalizer, msb.store)
```

Comment:
even this getHeader you don't need to define, as you can directly use GetMetaHeaderFromStorage
this wrapper does no extra processing on the data